docs/quality.md

```diff
@@ -1,3 +1,60 @@
 # Quality
 
-> @wip
+- Don't want to break this for driver, mongoose
+- Want input from these maintainers on establishing best practies, not to just
+  create more work
+- Result of this will be incorporated into our tooling,
+  [mj](https://github.com/rueckstiess/mj), for automated organization wide
+  checking and scaffolding for new projects
+
+## Testing
+
+- At the very least
+  - run mocha with one test that just tries to require your code
+  - travisci hook
+- For browser modules
+  - use [zuul](https://github.com/defunctzombie/zuul) + [saucelabs](https://saucelabs.com/)
```

Comment:
as far as I can tell they're pretty interchangeable. Karma isn't necessarily for angular, I use it for mongoose' browser tests too